client/paper/whisker.js

```diff
@@ -0,0 +1,171 @@
+import EventEmitter from 'events';
+
+export default class WhiskerFactory {
```

Comment:
Not sure what the point is of this factory? The code might be simpler by just giving each whisker a unique identifier and getting a unique canvas for each whisker. Then you can keep everything contained in `class Whisker`. Doesn't matter all that much, just thinking it might be simpler.

Reply:
I wanted to have just a single interval, but I agree that it would be simpler for each whisker to create its own interval. I'll change that.